shared-ui/components/footer/Footer.tsx

```diff
@@ -0,0 +1,69 @@
+import React, {useState} from 'react';
+import { FooterProps, TabInfo } from '../../lib/types';
+import {
+  StyledButtonContainer,
+  StyledButtonRow,
+  StyledEmailInput,
+  StyledFooter,
+  StyledFooterText,
+  StyledFooterTextLink,
+  StyledFooterTextContainer,
+  StyledHackBeanpotLogo,
+  StyledSocialMediaButtons,
+  StyledTab,
+  StyledTabContainer
+} from './Footer.styles';
+import HackBeanpotWhiteLogo from '../../images/hackbeanpot-white-logo.png';
+import Instagram from '../../images/social-icons/instagram.svg';
+import Facebook from '../../images/social-icons/facebook.svg';
+import Medium from '../../images/social-icons/medium.svg';
+import Twitter from '../../images/social-icons/twitter.svg';
+import PrimaryButton from '../primary-button/PrimaryButton';
+
+const Footer: React.FC<FooterProps> = ({ tabs }) => {
+  const [email, setEmail] = useState<string>('');
+
+  return (
+    <StyledFooter>
+      <StyledHackBeanpotLogo
+        src={HackBeanpotWhiteLogo}
+        alt='White HackBeanpot Logo'
+        width='191px'
+        height='56px'
+      />
+      <StyledTabContainer>
+        {tabs.map((tab: TabInfo) => (
+          <StyledTab>
+            {tab.name}
+          </StyledTab>
+        ))}
+      </StyledTabContainer>
+      <StyledFooterTextContainer>
+        <StyledFooterText>
+          HackBeanpot, Inc. is a registered 501(c)(3) organization.
+        </StyledFooterText>
+        <StyledFooterTextLink href="">
+          Code of Conduct
+        </StyledFooterTextLink>
+      </StyledFooterTextContainer>
+      <StyledButtonContainer>
+        <StyledEmailInput
+          placeholder='Enter your email'
+          value={email}
+          onChange={(e: React.FormEvent<HTMLInputElement>): void => setEmail(e.currentTarget.value)}
+        />
+        <StyledButtonRow>
+          <PrimaryButton btnText='Join our mailing list' btnLink='' />
+          <StyledSocialMediaButtons>
```

Comment:
the social media buttons look a tad bit large in my opinion. looking at the figma, it seems like the should be about 1/3 the height of the "join our mailing address" button when at full screen. 

Also, I think the font in the textbox and buttons should be Nunito Sans?

Lastly, there may need to be a little more space between the "mailing address" button and the social media buttons

current version:
<img width="680" alt="Screen Shot 2022-09-20 at 8 54 49 PM" src="https://user-images.githubusercontent.com/37753574/191390505-a2cf9e2d-fdfc-42b7-942e-bbfbd40f90ef.png">

figma:
<img width="798" alt="Screen Shot 2022-09-20 at 8 51 07 PM" src="https://user-images.githubusercontent.com/37753574/191390146-dbefabb7-1c32-4d05-8e68-a0440c1ae5c1.png">

Reply:
Updated global primary button style to use nunito sans